### ISSUE TITLE
Uncaught Error: Call to undefined method FreshRSS_Entry::feedId()

### ISSUE THREAD

**Issue by parcelcat** [printfuck/xExtension-Readable#3]
There is a bug that prevents FreshRSS from updating feeds when Readable is enabled:

> PHP Fatal error:  Uncaught Error: Call to undefined method FreshRSS_Entry::feedId() in /var/www/FreshRSS/extensions/xExtension-Readable/extension.php:22\nStack trace:\n#0 /var/www/FreshRSS/lib/Minz/ExtensionManager.php(337): ReadableExtension->fetchStuff()\n#1 /var/www/FreshRSS/lib/Minz/ExtensionManager.php(308): Minz_ExtensionManager::callOneToOne()\n#2 /var/www/FreshRSS/app/Controllers/feedController.php(424): Minz_ExtensionManager::callHook()\n#3 /var/www/FreshRSS/app/Controllers/feedController.php(769): FreshRSS_feed_Controller::actualizeFeed()\n#4 /var/www/FreshRSS/lib/Minz/Dispatcher.php(119): FreshRSS_feed_Controller->reloadAction()\n#5 /var/www/FreshRSS/lib/Minz/Dispatcher.php(48): Minz_Dispatcher->launchAction()\n#6 /var/www/FreshRSS/lib/Minz/FrontController.php(76): Minz_Dispatcher->run()\n#7 /var/www/FreshRSS/p/i/index.php(57): Minz_FrontController->run()\n#8 {main}\n  thrown in /var/www/FreshRSS/extensions/xExtension-Readable/extension.php on line 22

Thanks for creating this extension!

**Comment by printfuck** [comment]
I'm sorry for your inconveniance.

Seems like a recent update to FreshRss broke compatibility with all previous versions. 
This small workaround should be working with all versions for now.

Can you tell me, how you'd upgrade your addon? Since I'm testing and developing on my main instance, I don't use addon update features of FreshRSS and don't know how to go about that. Would be nice to hear, if there are any problems or complications.

**Comment by parcelcat** [comment]
@printfuck Thank you, the extension now works as expected!

Since I'm using the [official FreshRSS Docker container](https://hub.docker.com/r/freshrss/freshrss), I fetched the latest version of the extension with Git and then manually loaded it into the container.

1. Obtain the container ID of the FreshRSS instance:

```
sudo docker ps | grep freshrss
```

2. Delete the existing version of the Readable extension:

```
sudo docker exec -it _container ID_ /bin/bash
rm -rf extensions/xExtension-Readable
```

3. Load the newest version of the Readable extension into the Docker container:

```
sudo docker cp xExtension-Readable _container ID_:/var/www/FreshRSS/extensions/
```

The extension is now updated in FreshRSS. It would be nicer if FreshRSS had an extension installer/updater in the settings, instead of just a list of installed and available extensions, but it doesn't look like this feature is implemented yet.
